src/test/java/api/loans/LoanAPIPolicyTests.java

```diff
@@ -47,6 +48,7 @@ public LoanAPIPolicyTests() {
   }
 
   //TODO: Split into multiple tests
+  @Ignore
```

Comment:
Please add a note to ignore

Reply:
added

Reply:
Why was the test not updated to initialise those policies, rather than ignoring the test?

Reply:
@roman-barannyk, we should also create a story/task to fix the test if we're not going to fix it in this PR.

Reply:
I ignored this test because it has comment:   
//TODO: Split into multiple tests. 
The behavior of this test doesn't correspond the name and the most of the policies are not initialized. The purpose of this test is not clear. 
I would create a story to fix this test

Reply:
What do you think?

Reply:
@marc-johnson 
What do you think about this test? Should I create a story to fix and split it? How should it be split?

Reply:
It's been a long time since I made that TODO comment (and should've added more details).

A brief review of the test suggests to me that this behaviour is covered elsewhere, and this test should be deleted rather than split or amended.